### PR TITLE
Our Fancy Music Project (Caroline, Malin, & Elaine)

### PR DESCRIPTION
**Pull request by CoffeeDoodles**
Developers: Caroline Byström, Malin Gustafsson, Elaine Bergström

Link to my deployed project:
https://fancy-music-project.netlify.app/

My reflections on how this week's project turned out:
We found breaking down the project into pieces and pasting it on a organization site like Trello helped up all keep track of where we were in the project. We would probably like to try and get more separated components working the next time but are proud of what we did achieve with our current project. We stuck to our deadline and worked well together. go team!

Things I'd like to have clarified or explained in more detail:
We would like to discuss more about multiple components with multiple .map functions and just how to smoothly slap them together in the code. We'd also like to know how to substitute an '&' instead of commas for the red level requirements.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
 import React from 'react'
 import data from './data.json'
+import Album from './Components/Album.js'
+import Header from './Components/Header.js'
```
Comment (KarolCislowski) [inline code/src/App.js:4]: There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.

File: code/src/Components/Album.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+//image needs an URL
+const Album = (props) => {
+  console.log(props)
+    return (
+      <div className="album-card">
+          <div className="image-container">            
+              <img className="album-image" src={props.imgsrc}></img>          
+            <Buttons />
+          </div>
```
Comment (KarolCislowski) [inline code/src/Components/Album.js:12]: When your album title is longer then 1 line it makes your album photo break the grid on top. Your photos should be in line and your text areas should be ready for longer text.

image

File: code/src/Components/Album.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <Buttons />
+          </div>        
+        <div className="album-information-container">
+          <a className="url-link" href={props.url} target="_blank">
+            <h3 className="album-name">{props.name}</h3>
+          </a>
+        </div>
+
+        {props.artist.map((artist) => {
+          console.log(artist)
```
Comment (KarolCislowski) [inline code/src/Components/Album.js:20]: remember to remove console.logs  from your code after you finish debugging your code.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
This is a very nice project! You need to experiment a little more with your grid to make it align better. 🦄

File: code/src/App.js (hunk 1 above)
Comment (CoffeeDoodles) [reply code/src/App.js:4]: So we also need to import the style.css into the app.js file?

File: code/src/Components/Album.js (hunk 2 above)
Comment (CoffeeDoodles) [reply code/src/Components/Album.js:12]: I believe we've fixed this issue now! :)

**Review by KarolCislowski — APPROVED** [review]
Nice 😸

**Pull request closed without merging**